packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -303,6 +313,8 @@ class MessageList extends Widget<Properties> {
 
       if (author?.id === lastMessageGroupUserId && !isTimeoutExceeded && !shouldCreateDayHeader) {
         lastMessageGroup.renderMessage(message);
+        lastMessageGroup._updateIsLastOptionOfLastMessage(true);
```

Comment:
[excess template call]
That's not good that we we render last bubble twice: with `isLast=false` and then with `isLast=true`

Reply:
0e11dd012baad48fce62eeab905c60abc39aa9d5